src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -209,10 +209,15 @@ const ChatItemMyTextV2 = ({
   }, []);
 
   const handleTextContainerStyle = () => {
+    const isAttachment = attachments.length > 0;
+    const paddingStyle = {
+      paddingLeft: isAttachment ? BUBBLE_LEFT_PADDING_ATTACHMENT : BUBBLE_LEFT_PADDING,
+      paddingRight: isAttachment ? BUBBLE_RIGHT_PADDING_ATTACHMENT : BUBBLE_RIGHT_PADDING
+    };
     if (chatType === SIGNED) {
-      return [styles.containerSigned, styles.textContainer];
+      return [styles.containerSigned, styles.textContainer, paddingStyle];
     }
-    return [styles.containerAnon, styles.textContainer];
+    return [styles.containerAnon, styles.textContainer, paddingStyle];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This is a good refactor that takes into account whether there are attachments or not and adjusts the padding accordingly. It improves the flexibility of the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->